server/start.go

```diff
@@ -533,10 +533,6 @@ func startAPIServer(
 }
 
 func startTelemetry(cfg serverconfig.Config) (*telemetry.Metrics, error) {
-	if !cfg.Telemetry.Enabled {
```

Comment:
Now that `cfg.Telemetry.Enabled` is unused, I think we should remove it from the server config, template and update confix for v0.51